<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignInPage</class>
 <widget class="QWidget" name="SignInPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SignInPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
      <widget class="QLabel" name="welcomeLabel">
       <property name="minimumSize">
        <size>
         <width>178</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>178</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>Welcome to the Online Chat App!</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
      <widget class="QLabel" name="createAnAccountLabel">
       <property name="text">
        <string>Create a new account</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>52</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QLabel" name="userNotFoundLabel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <stylestrategy>PreferDefault</stylestrategy>
         <hintingpreference>PreferVerticalHinting</hintingpreference>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ContextMenuPolicy::ActionsContextMenu</enum>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Plain</enum>
       </property>
       <property name="text">
        <string>User alerady exist</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
      <widget class="QLineEdit" name="nameInput">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>28</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	background-color: rgba(0,0,0,0);
	border: none;
	border-bottom: 1px solid rgb(105, 118, 132);
}

QLineEdit:focus {
	background-color: rgba(0,0,0,0);
	border: none;
	border-bottom: 1px solid rgb(0, 255, 132);
}</string>
       </property>
       <property name="placeholderText">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
      <widget class="QLineEdit" name="loginInput">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>28</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	background-color: rgba(0,0,0,0);
	border: none;
	border-bottom: 1px solid rgb(105, 118, 132);
}

QLineEdit:focus {
	background-color: rgba(0,0,0,0);
	border: none;
	border-bottom: 1px solid rgb(0, 255, 132);
}</string>
       </property>
       <property name="placeholderText">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
      <widget class="QLineEdit" name="passwordInput">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>28</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	background-color: rgba(0,0,0,0);
	border: none;
	border-bottom: 1px solid rgb(105, 118, 132);
}

QLineEdit:focus {
	background-color: rgba(0,0,0,0);
	border: none;
	border-bottom: 1px solid rgb(0, 255, 132);
}</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
      <widget class="QPushButton" name="signInButton">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>28</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #2196F3;
    border: none;
    border-radius: 4px;
}

QPushButton:hover {
    background-color: #1976D2;
}

QPushButton:pressed {
    background-color: #0D47A1;
}

</string>
       </property>
       <property name="text">
        <string>Sign In</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
      <widget class="QPushButton" name="loginButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>Already have an account? Login in!</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
